script/Ciel/Day 6/06_01_CIEL06_1_2.txt

```diff
@@ -181,17 +188,23 @@ But as her older brother, I can't let her hear such a story. I would much rather
 [sha:8dc6f56686c12c23cb08f51187d905aa5b32dc9a]{
 -- Page 75, Offset 13354.
 -- 「そ、そう。最近知り合ったヤツなんだけど、一日だけでいいから観光名所を回りたいとか言い出して、気がつけば湾岸の埋め立て地あたりまで遠出したというか……」
-"Y-Yeah. I only met them recently, but they wanted to do some sightseeing around the famous places. Considering it was only for a day, they told me I should come along, and before I knew it, we had gone as far as the reclaimed land by the bay coast..."
+// 埋め立て地 can be either landfill (reclaimed land) or landfill (trash heap),
```

Comment:
that's "land fill" with a space, not "landfill" per the [Wikipedia](https://en.wikipedia.org/wiki/Land_reclamation). So yeah, this interpretation makes sense.